multinet/util.py

```diff
@@ -4,14 +4,25 @@
 
 from flask import Response
 
-from typing import Sequence, Any, Generator
+from typing import Sequence, Any, Generator, Dict
 
 from . import db
 from .errors import DatabaseNotLive, DecodeFailed
 
 TEST_DATA_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), "../test/data"))
 
 
+def filter_unwanted_keys(rows: Sequence[Dict]) -> Sequence[Dict]:
```

Comment:
This function actually returns a list, which as I mentioned elsewhere in the review, might be a bad idea. There should be a way to write this function to return a generator instead, which will help with unexpected memory blowup etc.